inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingService.java

```diff
@@ -100,15 +106,28 @@
         .asList("commonsmedia", "external-id", "globe-coordinate", "math", "monolingualtext",
             "quantity", "string", "url", "wikibase-property"));
 
-    private static final String WIKIDATA_PREFIX = "http://www.wikidata.org/entity/";
-    private static final String POS_VERB_PREFIX = "V";
-    private static final String POS_NOUN_PREFIX = "N";
-    private static final String POS_ADJECTIVE_PREFIX = "J";
-
     // A cache for candidates retrieved by fulltext-search
     private LoadingCache<CandidateCacheKey, Set<CandidateEntity>> candidateFullTextCache;
     private LoadingCache<SemanticSignatureCacheKey, SemanticSignature> semanticSignatureCache;
 
+    @Autowired
+    public ConceptLinkingService()
+    {
+
+    }
+
+    public ConceptLinkingService(KnowledgeBaseService aKbService,
+        EntityLinkingProperties aProperties)
+    {
+        kbService = aKbService;
+        properties = aProperties;
+        stopwordsFile = new File("${repository.path}/resources/stopwords-en.txt");
```

Comment:
The repository path is not substituted here. `${...}` does not work when used as a parameter to "File"